### PR TITLE
Perik #70 add notes entity to db

### PR DESCRIPTION
**Pull request by PerikK**

### CODE REVIEW

**Review by MrStashy — DISMISSED** [review]

**Review by FBagdeli — DISMISSED** [review]

File: prisma/migrations/20240722122140_add_note_model_update_user_model/migration.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+-- This is an empty migration.
```
Comment (Auenc) [inline prisma/migrations/20240722122140_add_note_model_update_user_model/migration.sql]: is this file needed? It's empty

**Review by Auenc — CHANGES_REQUESTED** [review]

File: prisma/migrations/20240722122140_add_note_model_update_user_model/migration.sql (hunk 1 above)
Comment (PerikK) [reply prisma/migrations/20240722122140_add_note_model_update_user_model/migration.sql]: It's vital!!! 
That said, I have no idea why it's there.
I ll take it out.

**Review by Auenc — APPROVED** [review]

**Review by MrStashy — APPROVED** [review]

**Review by FBagdeli — APPROVED** [review]
